src/content/development/contribution-guide/_index.en.md

```diff
@@ -234,6 +235,16 @@ During the week when the general availability release is performed, the next ver
 Additionally, a retrospective meeting for the last release cycle will be held.
 There will be no dedicated [test day](#test-days), as the release candidate has been tested and no changes have occurred since.
 
+##### Release Notes
+
+[Release notes] should be published for each GA version.
+All new features and bug fixes of the release should be labeled with `release-note-needed` and have a relevant release note
+documented as part of the PR. The content of the release-note should be in a file under `release-notes`.
```

Comment:
```suggestion
documented as part of the PR. Release notes submitted with PRs should be written in individual files in the `/release-notes` directory of the relevant repository.
```

Should we stipulate something about the naming of release note files (`NNN-description`)? Should we stipulate something about formatting (txt vs yaml with some well-known scheme?)

Reply:
For now I've been writing them as plain text files, with a sensible name.

I think we need to see how that works out and refine from there in the future.

Reply:
> A monotonically-increasing prefix would be useful to make it obvious which release notes are relevant (or at least, which release note snippets are so old they can’t be relevant, without looking at the commit metadata where they were introduced; this would also allow easy clean-ups). If we make it the issue number, that would also ensure that release note items correspond to an actual issue (bug or enhancement request) ;-).

Possibly, but issue number is a bit problematic since the issue might be something in enhancements, and thus not really telling.

I'm not sure we expect anyone to read them from here though, so not sure how valuable it would be to have a way to identify old notes.

> Regarding the content, I think plain text is fine, as @mkolesnik has been doing recently. Perhaps we can document it as being Markdown snippets.

We could have them be `.md` files and then they'll be linted automatically (though we'll probably want to link them with `MD041/first-line-heading/first-line-h1` off).

Reply:
> I'm not sure we expect anyone to read them from here though, so not sure how valuable it would be to have a way to identify old notes.

I don’t expect anyone to read them from here either; all I was after was avoiding an ever-growing directory of files. We could hand-wave it all away by saying automation will deal with that, but if we can have an easy-to-use human-compatible approach that enables that too, all the better IMO. A date-based prefix would do the trick too.

Reply:
Yes I think date prefix YYYYMMDD would make sense and help with your expectation too